.storybook/docs-root.css

```diff
@@ -45,13 +45,13 @@
   border-radius: 24px;
 }
 
-#storybook-docs .sbdocs:not(.sbdocs-preview) hr {
+#storybook-docs .sbdocs hr:not(.sbdocs-preview hr) {
```

Comment:
these negations add even more complexity, can we  aggregate and simplify all of these ?

eg
```css
#storybook-docs .sbdocs:not(.sbdocs-preview) p {}

#storybook-docs .sbdocs:not(.sbdocs-preview) hr {}

#storybook-docs .sbdocs:not(.sbdocs-preview) h2 {}

#storybook-docs .sbdocs:not(.sbdocs-preview) h2 code {}
```

Reply:
I tried that, and unfortunately it doesn't work as expected as there multiple elements with the `sbdocs` class nested. Here's an example:

![image](https://github.com/user-attachments/assets/dfcaf17d-caa9-45bb-955a-17189781b635)

Reply:
sad to see this. the tax of tweaking non public APIs... ty for clarification